AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Sync/SubscriptionSync/ModelReconciliationDeleteTests.swift

```diff
@@ -57,9 +59,9 @@ class ModelReconciliationDeleteTests: SyncEngineTestBase {
         }
         wait(for: [expectationListener], timeout: 2.0)
 
-        guard let listener = listenerFromRequest else {
-            XCTFail("Incoming responder didn't set up listener")
-            return
+        guard let valueListener = valueListenerFromRequest else {
+                XCTFail("Incoming responder didn't set up listeners")
```

Comment:
isn't there just one listener? not sure why this was changed from listener to listeners

Reply:
Derp. That was there when I thought I needed to `guard` for both the value and completion listeners. Will change back.